### PR TITLE
Monthly statistics ITT

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

We are now in a new cycle, we need to ensure we continue to surface the data we surfaced for our users last cycle
stakeholders make use of this data for modelling

### CODE REVIEW

**Comment by tomas-stefano** [comment]
[Monthly report: Make sure we have a new page for 2023 cycle rolling count, and historical page for 2022 cycle](https://trello.com/c/uXYV1hPy/904-monthly-report-make-sure-we-have-a-new-page-for-2023-cycle-rolling-count-and-historical-page-for-2022-cycle)

File: app/controllers/publications/monthly_statistics_controller.rb [hunk 1]
```diff
@@ -29,16 +26,21 @@ def calculate_download_sizes(report)
...
-    def temporarily_unavailable; end
-
   private
 
     def current_report
-      if params[:month].present?
-        MonthlyStatisticsTimetable.current_report_at(Date.parse("#{params[:month]}-01"))
+      return MonthlyStatisticsTimetable.report_for_current_period if params[:month].blank?
+
+      if itt_match.present?
```
Comment (carlosmartinez) [inline app/controllers/publications/monthly_statistics_controller.rb:34]: how about having a separate route:
```
get '/monthly-statistics/ITT(:year)' => 'monthly_statistics#show', as: :monthly_report_itt
```
and then this condition is `if params[:year].present?` and we don't need to bother with regex matching here.

Comment (carlosmartinez) [reply app/controllers/publications/monthly_statistics_controller.rb:34]: and this action becomes a bit simpler:

```
    def current_report
      if params[:month].present?
        MonthlyStatisticsTimetable.current_report_at(Date.parse("#{params[:month]}-01"))
      elsif params[:year].present?
        MonthlyStatisticsTimetable.report_for_latest_in_cycle(params[:year].to_i)
      else
        MonthlyStatisticsTimetable.report_for_current_period
      end
    end
```

Comment (tomas-stefano) [reply app/controllers/publications/monthly_statistics_controller.rb]: Done ♻️

**Review by carlosmartinez — APPROVED** [review]

**Pull request merged**